centreon/packages/ui/src/Dialog/index.tsx

```diff
@@ -110,7 +100,7 @@ const Dialog = ({
         {onCancel && (
           <Button
             color="primary"
-            data-testid={dataTestIdCanceledButton}
+            data-testid={t(labelCancel as string)}
```

Comment:
the test id should be a constant (no translation as it would change according to the language)